### ISSUE TITLE
[Idea] Check the pending authorization list backwards

### ISSUE THREAD

**Issue by MarcoStuurman** [letsdebug/letsdebug-toolkit#2]
Maybe it's a good idea to check the pending auth list backwards limiting at 300 pending authorizations found. This reduces load at Let's Encrypt and saves time for the checker.

**Comment by alexzorin** [comment]
Good idea. The `grep -R` method of collecting logs will have to change, though, since that basically produces a random order, forcing us to do a full scan.

**Comment by MarcoStuurman** [comment]
I think the challenge id is sequential (e.g. 569414841 569414842). After indexing the id’s it could be ordered high to low
